src/util.ts

```diff
@@ -101,8 +101,15 @@ export var getProp = maybePromise(function(obj, prop) {
   return obj[prop];
 });
 
-export function ensureExtension(fileName: string, extension: string) {
-  return fileName + '.' + extension;
+export function ensureExtension(fileName: string, extension: string): string {
```

Comment:
[path.extname](https://nodejs.org/api/path.html#path_path_extname_p) might be useful here